engine/bsql_engine/io/cio.pxd

```diff
@@ -203,10 +203,7 @@ cdef extern from "../include/io/io.h":
         json_read_arg jsonReaderArg
         csv_read_arg csvReaderArg
     cdef struct TableSchema:
-        # TODO: TableSchema will be refactorized
-        # BlazingTableView blazingTableView
-        vector[gdf_column_ptr] columns
-        vector[string]  names
+        BlazingTableView blazingTableView
```

Comment:
If I am not mistaken we used to use a vector of gdf_column to contain the schema, where the columns would hold the types and it would also be the gdf data for a gdfTable. But I think now we want to have a vector of types and a vector of names to define the schema and have a separate BlazingTableView that is used ONLY for gdfTables. And the rest of the code should be updated to follow this new pattern

Reply:
Ohh. Ok that should change some lines of code ...